scripts/asm-installer/asm_vm

```diff
@@ -1110,6 +1110,23 @@ this project, run:
   --project="${PROJECT_ID}"
 EOF
     fi
+
+    info "Verifying GCE image..."
+    local SOURCE_IMAGE_PATH="$(echo "$(retry 2 run gcloud compute instance-templates \
+      describe "${SOURCE_INSTANCE_TEMPLATE}" \
+      --format=json \
+      --project="${PROJECT_ID}")" \
+    | jq '.properties.disks[] | select(.initializeParams.sourceImage != null) | .initializeParams.sourceImage')"
```

Comment:
Does this solve @jasonwzm 's comment in https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/pull/536#discussion_r592822445?

Reply:
What if each disk has a source image? This jq outputs all the source image paths.

Reply:
updated